spec/rails_helper.rb

```diff
@@ -9,34 +7,16 @@
 require File.join(File.dirname(__FILE__), 'support/valid_attribute')
 require File.join(File.dirname(__FILE__), 'support/factory_girl')
 require 'capybara/rspec'
-# Add additional requires below this line. Rails is not loaded until this point!
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.